src/pages/get-started.tsx

```diff
@@ -26,19 +26,20 @@ const GetHelpSection = () => {
             </p>
             {/* prettier-ignore */}
             <CodeBlock language="bash" showLineNumbers>
-                $ podman --help # get a list of all commands {'\n'}
-                $ podman subcommand --help # get info on a command{' '}
-                {'\n'}
+                $ man podman {'\n'} 
+                $ man podman subcommand {'\n'}
               </CodeBlock>
+           
           </div>
           <div className="container grid grid-cols-1 gap-y-4 lg:grid-cols-2 lg:gap-y-0">
             <p className="max-w-sm text-white">
               To get some help and find out how Podman is working, you can use the help.
             </p>
             {/* prettier-ignore */}
             <CodeBlock language="bash" showLineNumbers>
-                $ man podman {'\n'} 
-                $ man podman subcommand {'\n'}
+                $ podman --help # get a list of all commands {'\n'}
+                $ podman subcommand --help # get info on a command{' '}
+                {'\n'}
```

Comment:
I think, but don't know, that this has to be at the end fo the previous line.  @cgiradkar  do you know?

Reply:
@TomSweeneyRedHat  I tried to modify the code as: 
```
$ podman --help # get a list of all commands {'\n'}
$ podman subcommand --help # get info on a command{'\n'}
```

and the result is the same (please see the screenshot below). I tried to build it locally (`yarn build  & yarn serve). 
![Screenshot from 2023-11-06 11-17-21](https://github.com/containers/podman.io/assets/141641852/54b9bcca-287a-43b7-9167-6f18c406f977)

Reply:
> I think, but don't know, that this has to be at the end fo the previous line. @cgiradkar do you know?

I agree, the extra carriage returns () are not needed hence should be removed.